plugins/modules/files/iso_get_files.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2022, Ansible Project
+# Copyright (c) 2022, VMware, Inc. All Rights Reserved.
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: iso_get_files
+short_description: Fetch files from ISO
+description:
+  - This module is used to get specific files from ISO.
```

Comment:
What I'm not sure is: how is this module related to the existing `iso_extract` module?

Reply:
iso_extract will extract the whole iso content. It will consume time and space if the ISO have big size.

Background:
For some OS ISO, we need to modify the the original iso in order to support unatended installation
the easiest way is to modify the original iso with API but not "extract the ISO, modify file, regenerate new ISO"
 
Thanks
Yuhua Zou

Reply:
From 'iso_extract's documentation, it requires a list of files to be specified that will be extracted. It does not seem to extract the whole ISO. How is this different from your module?